packages/node_modules/@ciscospark/internal-plugin-metrics/src/config.js

```diff
@@ -7,7 +7,7 @@ import {inBrowser} from '@ciscospark/common';
 export default {
   device: {
     preDiscoveryServices: {
-      metricsServiceUrl: process.env.METRICS_SERVICE_URL || 'https://metrics-a.wbx2.com/metrics/api/v1'
+      metrics: process.env.METRICS_SERVICE_URL || 'https://metrics-a.wbx2.com/metrics/api/v1'
```

Comment:
I'd rather not change these from other plugins. If we decide the `ServiceUrl` suffix is too verbose, we should change it in the repo in another PR.

Reply:
@mattnorris Thanks for your comment. I would rather not change this suffix either, haha. The reason I need to change this service name is that service side changed the key(remove the suffix ServiceUrl) of the map and I have to change some functions in wdm module to make it compatible with the new map. Since the metricsServiceUrl  in preDiscoveryServices in hard coded, I think we could probably change it to make it consistent with the message on server side. Any thought on this?

Reply:
@stanjiawang  Thank you for clarifying. I missed this point in the description.

Reply:
@mattnorris Sure! I will try to replace all the service name without "ServiceUrl" in the repo. Thanks!